src/main/java/com/mongodb/jdbc/MongoSQLDatabaseMetaData.java

```diff
@@ -1141,21 +1145,21 @@ CASE_SENSITIVE, new BsonBoolean(BSON_BINDATA.getCaseSensitivity())),
                         new BsonElement(TYPE_NAME, new BsonString(BSON_BOOL.getBsonName())),
                         new BsonElement(DATA_TYPE, new BsonInt32(Types.BIT)),
                         new BsonElement(PRECISION, asBsonIntOrNull(BSON_BOOL.getPrecision())),
-                        new BsonElement(LITERAL_PREFIX, n),
-                        new BsonElement(LITERAL_SUFFIX, n),
-                        new BsonElement(CREATE_PARAMS, n),
+                        new BsonElement(LITERAL_PREFIX, BSON_NULL_VALUE),
+                        new BsonElement(LITERAL_SUFFIX, BSON_NULL_VALUE),
+                        new BsonElement(CREATE_PARAMS, BSON_NULL_VALUE),
                         new BsonElement(NULLABLE, new BsonInt32(ResultSetMetaData.columnNullable)),
                         new BsonElement(
                                 CASE_SENSITIVE, new BsonBoolean(BSON_BOOL.getCaseSensitivity())),
                         new BsonElement(SEARCHABLE, new BsonInt32(typeSearchable)),
```

Comment:
We can also refactor these `new BsonInt32(typeSearchable))` calls as typeSearchable is a final int value `public static final int typeSearchable = 3`.